magefiles/installation/install.go

```diff
@@ -105,6 +105,29 @@ func NewAppStudioInstallController() (*InstallAppStudio, error) {
 		LocalForkName:                    DEFAULT_LOCAL_FORK_NAME,
 		LocalGithubForkOrganization:      utils.GetEnv("MY_GITHUB_ORG", DEFAULT_LOCAL_FORK_ORGANIZATION),
 		QuayToken:                        utils.GetEnv("QUAY_TOKEN", ""),
+		DefaultImageQuayOrg:              utils.GetEnv("DEFAULT_QUAY_ORG", DEFAULT_E2E_QUAY_ORG),
+		DefaultImageQuayOrgOAuth2Token:   utils.GetEnv("DEFAULT_QUAY_ORG_TOKEN", ""),
+		DefaultImageTagExpiration:        utils.GetEnv(constants.IMAGE_TAG_EXPIRATION_ENV, constants.DefaultImageTagExpiration),
+	}, nil
+}
+
+func NewAppStudioInstallControllerUpgrade(infraFork string, infraBranch string) (*InstallAppStudio, error) {
+	cwd, _ := os.Getwd()
+	k8sClient, err := kubeCl.NewAdminKubernetesClient()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &InstallAppStudio{
+		KubernetesClient:                 k8sClient,
+		TmpDirectory:                     DEFAULT_TMP_DIR,
+		InfraDeploymentsCloneDir:         fmt.Sprintf("%s/%s/infra-deployments", cwd, DEFAULT_TMP_DIR),
+		InfraDeploymentsBranch:           infraBranch,
+		InfraDeploymentsOrganizationName: infraFork,
+		LocalForkName:                    DEFAULT_LOCAL_FORK_NAME,
+		LocalGithubForkOrganization:      utils.GetEnv("MY_GITHUB_ORG", DEFAULT_LOCAL_FORK_ORGANIZATION),
+		QuayToken:                        utils.GetEnv("QUAY_TOKEN", ""),
```

Comment:
@jkopriva I guess these are unrelated changes.

also [these](https://github.com/redhat-appstudio/e2e-tests/pull/1028/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR1001) and [these](https://github.com/redhat-appstudio/e2e-tests/pull/1028/files#diff-8d3f02a956e3c919ac5ea21a43aa554db0d763ffa49bac99f9f294d88e612d59R384)